static/src/javascripts/projects/common/views/experiments/quick-survey.html

```diff
@@ -0,0 +1,92 @@
+<style>
```

Comment:
By inlining it, we can avoid having to load an extra css file just for this fragment. Is there another way we could do this ?